### ISSUE TITLE
Indented Paragraphs

### ISSUE THREAD

**Issue by Kensai2000** [writemonkey/wm3#52]
Try as I might, I cannot get normal text to present as paragraphs with an intended first line. Block paragraphs separated by extra carriage returns is fine for blogging or email, but for writing fiction, I just can't get used to it. I know other apps have gotten around Markdown using TAB to indicate code blocks by simply having an "indent first line" setting that only effects "normal" text (rather than headings and everything else), but not finding one present in the  settings file, and failing to figure out how to force a theme to apply such formatting, I figure I'll need to request it here.

**Comment by Negreuj** [comment]
I'd also dearly support the idea of being able to have **first line of paragraph indented**, but also a way to manipulate **distance between paragraphs**. Both are features in WM2 - and I haven't been able to find them in WM3.

**Comment by JackPolo** [comment]
I have to agree. **First line paragraph indents** and **paragraph distance options** are a must for fiction/non-fiction writers like myself. So happy WriteMonkey is functional on Linux now and those two options would seal the deal for this amazing program.

**Comment by maxmv** [comment]
Must-have feature

**Comment by writemonkey** [comment]
This one is more difficult than it seems, but it will happen.
i.

**Comment by akadormouse** [comment]
Is it still on the todo list?
It's the most important absent feature for me.
My actual preference would be new line indents (I don't want to have to press Enter twice each time for a new paragraph, I don't want to see a blank line between 'paragraphs', and it's easier to convert all the lines to paragraphs on export).
